pkg/manager/internal/controllers/servicecluster_controller.go

```diff
@@ -35,8 +36,9 @@ import (
 
 type ServiceClusterReconciler struct {
 	client.Client
-	Log    logr.Logger
-	Scheme *runtime.Scheme
+	Log                  logr.Logger
+	Scheme               *runtime.Scheme
+	MonitorGraceDuration time.Duration
```

Comment:
```suggestion
	MonitorGracePeriod time.Duration
```
Please be consistent with period or duration